### DISCUSSION TITLE
Lesson 6 import script error

### DISCUSSION THREAD

**Discussion by ArtificialSelection** [Cyfrin/foundry-full-course-cu#532] [General]
I am getting this import error for script. I am not sure why. 
![error](https://github.com/Cyfrin/foundry-full-course-f23/assets/85599175/3f133026-268b-4571-ace6-4dacf8c6d10e)

All files are arranged properly
![error2](https://github.com/Cyfrin/foundry-full-course-f23/assets/85599175/e1f8fc00-8642-416d-8b1e-c518c697ac37)

Even though it is throwing an error but files are compiling.
![error3](https://github.com/Cyfrin/foundry-full-course-f23/assets/85599175/76d2c4a7-789b-4047-9a97-368669c31cca)

Still I would like to get rid of this error could someone please let me know how to get rid of it.
Thank you

**Comment by Naman1729** [comment]
Hey @ArtificialSelection , Run this command in your terminal :
```
foundryup
```

**Reply by ArtificialSelection** [reply]
Hello @Naman1729, error still persist

**Reply by Naman1729** [reply]
you can do this to avoid error :
```solidity
import {Script} from "../lib/forge-std/src/Script.sol";
```
Can you paste your foundry.toml here .

**Reply by alfheimrShiven** [reply]
Hey @Naman1729 curious here, how will running `foundryup` help resolve this?

**Reply by shikhar229169** [reply]
I think in command line you are in different directory.

If that not so, then try restarting the vs code and try again. I also faced that several times and it doesn't compile (but in your case its compiling successfully). Restarting vs code fixed it for me.

**Reply by ArtificialSelection** [reply]
Hello @Naman1729, the error got solved with the new import line suggestion.
Thank you

**Reply by rroyo** [reply]
Hello!
Had the same problem, specifying the full path solves the error.
But it doesn't answer why this is happening. In my case it seems VSC needed a restart.
I hadn't closed it while installing Foundry. A quick close and run again, solved the problem for me.
